locales/en/default.json

```diff
@@ -1,6 +1,8 @@
 {
-  "phase_banner": "This is a new service",
-  "phase_banner_with_feedback": "This is a new service – your <a class=\"govuk-link\" href=\"{{url}}\">feedback</a> will help us to improve it.",
   "age": "Age",
-  "age_with_date_of_birth": "{{date_of_birth}} (Age {{age}})"
+  "age_with_date_of_birth": "{{date_of_birth}} (Age {{age}})",
+  "primary_navigation": {
+    "dashboard_police": "Upcoming moves",
```

Comment:
We should probably use a key that relates to the content of the translation rather than confusing them by using people's roles. "Upcoming moves" isn't also just tied to "Police". Some prison users, STC and SCH users will see this item.